src/Symfony/Component/Validator/Resources/translations/validators.en.xlf

```diff
@@ -192,7 +192,7 @@
             </trans-unit>
             <trans-unit id="51">
                 <source>No temporary folder was configured in php.ini.</source>
-                <target>No temporary folder was configured in php.ini, or the configured folder does not exist.</target>
+                <target>No temporary folder was configured in php.ini or the configured folder does not exist.</target>
```

Comment:
Documentation Standards indicate that Serial (Oxford) Commas should be avoided.

Reply:
https://en.wikipedia.org/wiki/Serial_comma

> . British English allows constructions with or without this comma,[6] while in American English it is common and even often considered mandatory.

> Most British style guides do not mandate its use. The Economist Style Guide notes that most British writers only use it where necessary to avoid ambiguity.[11] However, a few British style guides mandate it, most notably The Oxford Style Manual.

So my undertstanding is "always use a serial comma" is what should be avoided, while it should be applied when necessary to avoid ambiguity.

Here the serial comma was valid IMHO. See also https://www.grammarly.com/blog/comma-before-or/

Reply:
I'm not saying it's grammatically incorrect. I've just noticed, that `or` is used in 17 different translation items and only once with comma. Plus, in this case, the context would not be ambiguos, if the comma was avoided.

Reply:
> used in 17 different translation items and only once with comma

grammar still applies per case ... not sure we aim for consistency rather than improving translations (per case).

in this  case i think `or the configured folder does not exist` is an independent clause, thus needs a serial comma.

We should avoid the serial comma, if the the message were like `No temporary folder was configured in php.ini or other-file.ini` (a dependent clause).

Reply:
I agree with @ro0NL

Reply:
I would also leave it as is, at least until a native English speaking person complains ;)

Reply:
Done